config/sync/core.entity_view_display.node.stanford_opportunity.default.yml

```diff
@@ -155,14 +197,14 @@ content:
       timezone_override: ''
       format_type: medium
     third_party_settings: {  }
-    weight: 114
+    weight: 7
     region: content
   su_opp_prerequisites:
     type: text_default
     label: above
     settings: {  }
     third_party_settings: {  }
-    weight: 118
+    weight: 8
     region: content
   su_opp_source:
```

Comment:
I'm unsure about the ordering of the External Link `su_opp_source` and Sponsor `su_opp_sponsor` 🤔

Reply:
The External link should not be displayed (please hide on the display. Only used for Rabbit hole.
I think the Sponsor can go below the page title.